src/opm/parser/eclipse/Deck/DeckItem.cpp

```diff
@@ -410,15 +425,20 @@ bool DeckItem::equal(const DeckItem& other, bool cmp_default, bool cmp_numeric)
         break;
     case type_tag::fdouble:
         if (cmp_numeric) {
-            const std::vector<double>& this_data = this->dval;
-            const std::vector<double>& other_data = other.dval;
+            const auto& this_data = this->getData<double>();
+            const auto& other_data = this->getData<double>();
```

Comment:
This still looks wrong?

Reply:
![image](https://user-images.githubusercontent.com/1321665/65874282-bcd8ef80-e385-11e9-9e17-d38658a1da8a.png)